src/_posts/databases/postgresql/2000-01-01-accessing.md

```diff
@@ -0,0 +1,238 @@
+---
+title: Accessing Your Scalingo for PostgreSQL® Addon
+nav: Accessing
+modified_at: 2023-12-01 00:00:00
+tags: databases postgresql addon
+index: 6
+---
+
+
+Remotely accessing your Scalingo for PostgreSQL® database can sometimes be
+useful, for example, to conduct investigations, to check or compute data
+locally, to dump the database content,...
+
+We provide several ways to access your database remotely. Chosing one mostly
+depends on your needs and preferences.
+
+## Using the Interactive Remote Console
+
+1. Make sure you have correctly [setup the Scalingo Command Line tool]({% post_url platform/cli/2000-01-01-start %})
+2. From the command line, open a console for your Scalingo for PostgreSQL®
+   addon:
+   ```bash
+   scalingo --app my-app pgsql-console
+   ```
+   The output should look like this:
+   ```bash
+   -----> Starting container one-off-7872  Done in 0.479 seconds
+   -----> Connecting to container [one-off-7872]...  
+   -----> Process 'pgsql-console my_app_4553' is starting...  
+
+   Hello from bashrc.
+   ---> Download and extract the database CLI
+   ---> Database CLI installed:
+   psql (PostgreSQL) 14.6
+   psql (14.6, server 14.8 (Debian 14.8-1.pgdg110+1))
+   SSL connection (protocol: TLSv1.3, cipher: TLS_AES_256_GCM_SHA384, bits: 256, compression: off)
+   Type "help" for help.
+
+   my_app_4553=>
+   ```
+3. To exit, type the following command from the PostgreSQL® console, or use the
+   keyboard shortcut `CTRL`+`D`:
+   ```bash
+   \q
+   ```
+
+
+## Using a One-Off
+
+1. [Start a one-off container]({% post_url platform/app/2000-01-01-tasks %})
+2. From the command line, use our `dbclient-fetcher` helper to download and
+   install a bunch of PostgreSQL® tools:
+   ```bash
+   dbclient-fetcher pgsql
+   ```
+   The output should look like this:
+   ```bash
+   ---> Download and extract the database CLI
+   ---> Database CLI installed:
+   psql (PostgreSQL) 14.6
+   ```
+3. If you ever need a specific version, just add it as a second parameter:
+   ```bash
+   dbclient-fetcher pgsql 13
+   ```
+   The output should look like this:
+   ```bash
+   ---> Download and extract the database CLI
+   ---> Database CLI installed:
+   psql (PostgreSQL) 13.11
+   ```
+
+The helper installs the following tools:
+- `psql`
+- `pg_basebackup`
+- `pg_controldata`
+- `pg_dump`
+- `pg_isready`
+- `pg_recvlogical`
+- `pg_restore`
+- `pg_test_fsync`
+- `pg_upgrade`
+- `pg_archivecleanup`
+- `pg_config`
+- `pg_ctl`
+- `pg_dumpall`
+- `pg_receivewal`
+- `pg_resetwal`
+- `pg_rewind`
+- `pg_test_timing`
+- `pg_waldump`
+
+
+## Using Third Party Tools
+
+By default, and for security reasons, your Scalingo for PostgreSQL® database is
+not directly accessible from the Internet.
```

Comment:
suggestion: I would add a first sentence to explain what this means to use a third party tool. Before explaining the fact that our databases are not reachable from the internet.

Reply:
Done in https://github.com/Scalingo/documentation/pull/2389/commits/b672481fa4f6324e041dc3b7d3293acd094b9c46